checks/all_checks.go

```diff
@@ -14,11 +14,13 @@
 
 package checks
 
+import "strings"
```

Comment:
Can you please combine these imports?
```
import(
"strings"
"github.com/ossf/scorecard/checker")
```

This will get rid of the lint failures.